en/config_rover/mecanum.md

```diff
@@ -0,0 +1,298 @@
+# Configuration/Tuning (Mecanum Rover)
+
+This topic provides a step-by-step guide for setting up your [Mecanum rover](../frames_rover/mecanum.md).
+Successive steps enable [drive modes](../flight_modes_rover/mecanum.md) with more autopilot support and features.
+
+::: warning
+Each step is dependent on the previous steps having been completed.
+Modes will only work properly if the preceding modes have been configured.
+:::
+
+## Basic Setup
+
+To start using the mecanum rover:
+
+1. Enable Rover support by flashing the [PX4 rover build](../frames_rover/index.md#flashing-the-rover-build) onto your flight controller.
+   Note that this is a special build that contains rover-specific modules.
+
+2. In the [Airframe](../config/airframe.md) configuration select _Generic Rover Mecanum_ frame:
+
+   ![QGC screenshot showing selection of the airframe 'Generic Rover Mecanum'](../../assets/config/airframe/airframe_generic_rover_mecanum.png)
+
+   Select the **Apply and Restart** button.
+
+   ::: info
+   If this airframe does not show up in the UI, it can alternatively be selected by setting the [SYS_AUTOSTART](../advanced_config/parameter_reference.md#SYS_AUTOSTART) parameter to `52000`.
+   :::
+
+3. Use [Actuators Configuration & Testing](../config/actuators.md) to map the motor functions to flight controller outputs.
```

Comment:
Is there anything special about this? I mean is it worth having a screenshot or two of a standard setup, or it is trivially easy for anyone to set up reading existing docs?

Reply:
Great. Sometimes it is worth showing an image for this kind of thing to show how trivial it is. Also helps answer questions like "what sort of ESCs should I use". For an MC that is ideally D-Shot, whereas for an FW it is often PWM.